backend/src/routes/books.ts

```
@@ -0,0 +1,116 @@
+ * @desc 책 추가
+ * @access Admin
+ */
+router.post(
+  '/',
+  async (
+    req: Request<{}, {}, { title: string; author: string }>,
+    res: Response
+  ) => {
+    try {
+      const book = new Book(req.body)
+      const newBook = await book.save()
```

Biome (1.9.4)

[error] Don't use '{}' as a type.

Prefer explicitly define the object shape. '{}' means "any non-nullable value".

(lint/complexity/noBannedTypes)